android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetView.java

```diff
@@ -184,10 +186,14 @@ public static void onHostPause(Activity activity) {
      * <tt>Activity.onResume</tt> so we can do the required internal processing.
      *
      * @param activity - <tt>Activity</tt> being resumed.
+     * @param backHandler - a <tt>DefaultHardwareBackBtnHandler</tt> which
+     * will handle the back button press in case there's nothing to handle on
+     * the JS side.
      */
-    public static void onHostResume(Activity activity) {
+    public static void onHostResume(Activity activity,
+                                    DefaultHardwareBackBtnHandler backHandler) {
```

Comment:
We talked today in person and I did take care of my comments in a subsequent commit but here's my review for the sake of completeness:

The above constitutes a breakage of the public API of `JitsiMeetView`. It can be avoided by preserving the old method, overloading it with the new one, and forwarding the old method to the new one with a `DefaultHardwareBackBtnHandler` implementation common to `JitsiMeetView` and `JitsiMeetActivity`.